src/actionExecutor.ts

```diff
@@ -39,7 +39,10 @@ export class ActionExecutor {
     let result: Result<any, Error>;
 
     const triggerResult = await this.triggerRequestFromAction(action);
-    logger.trace(`Response from action: ${JSON.stringify(triggerResult)}`);
+    logger.log(
+      "trace",
+      `Response from action: ${JSON.stringify(triggerResult)}`
```

Comment:
```suggestion
      "Response from action:",
       triggerResult`
```

We discussed that in the past, that we should pass the object directly to allow stackdriver to actually recognize the object and allow us to do research on such object properties. Is that still valid?

See the following quote in the original issue:
> log4js is more powerful than debug and supports arbitrary parameters to its log-fn calls: https://github.com/log4js-node/log4js-node/blob/master/examples/example.js#L38